questions/Gallery/AstOfCProgram/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "uuid": "34292487-5019-47ab-8175-49e9dada8790",
+    "title": "Sale Price",
+    "topic": "Variables",
+    "singleVariant": true,
+    "tags": [ "jgfoster" ],
+    "type": "v3",
+    "gradingMethod": "External",
+    "externalGradingOptions": {
+        "enabled": true,
+        "image": "prairielearn/grader-c",
+        "entrypoint": "/grade/tests/test.sh",
+        "serverFilesCourse": ["clang/"]
```

Comment:
This line and the serverFilesCourse content become unnecessary once https://github.com/PrairieLearn/PrairieLearn/pull/11913 is handled. Also, the entrypoint can become the Python file again if the path does not need to be updated.